packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 12.0.0
+
+* Adds error handling on Flutter Api methods.
```

Comment:
Aren't there breaking changes to at least Swift and Kotlin here that aren't mentioned?